src/components/navigation/navigation.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Documentation | MintUI</title>
+    <link rel="stylesheet" href="/src/css/root.css">
+    <link rel="stylesheet" href="/src/css/header.css">
+    <link rel="stylesheet" href="/src/css/docs.css">
+    <link rel="stylesheet" href="/src/css/utility.css">
+    <link rel="stylesheet" href="/src/components/navigation/navigation.css">
+    <script src="https://kit.fontawesome.com/8f32ce5d5f.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header class="header-container">
+        <nav class="header-content">
+            <a class="header-title" href=""><span>MINT</span>UI</a>
+            <div class="header-navigation">
+            <a href="/index.html">Home</a>
+            <a href="/src/pages/installation.html">Installation</a>
+            <a href="/src/components/avatars/avatars.html" class="link-active">Docs</a>
+            </div> 
+        </nav>
+    </header> 
+    <section class="docs-container">
+        <div class="docs-sidebar-container">
+            <div class="sidebar-content">
+                <div class="sidebar-title">Components</div>
+                <div class="sidebar-contents">
+                    <ul>
+                        <li>
+                            <a href="/src/components/avatars/avatars.html">Avatar</a> 
+                        </li>
+                        <li>
+                            <a href="/src/components/alerts/alerts.html">Alert</a> 
+                        </li>
+                        <li>
+                            <a href="/src/components/badges/badges.html">Badge</a> 
+                        </li>
+                        <li>
+                            <a href="/src/components/buttons/buttons.html">Button</a> 
+                        </li>
+                        <li>
+                            <a href="">Card</a> 
+                        </li>
+                        <li>
```

Comment:
add card link here

Reply:
Card component is done in a separate branch, will add once done